NEWS

```diff
@@ -437,41 +437,44 @@ CHANGES WITH 248:
-        Lucas Werkmeister, Luka Kudra, Luna Jernberg, Marc-André Lureau, Martin
-        Wilck, Matthias Klumpp, Matt Turner, Michael Marley, Michal Fabik,
-        Michał Kopeć, Michal Koutný, Michal Sekletár, Mike Gilbert, milovlad,
-        moson-mo, Nick, nihilix-melix, Oğuz Ersen, Ondrej Mosnacek, pali, Pavel
-        Hrdina, Pavel Sapezhko, Perry Yuan, Peter Hutterer, Pierre Dubouilh,
-        Piotr Drąg, Richard Laager, rnhmjoj, RussianNeuroMancer, Sebastiaan van
-        Stijn, Sergey Bugaev, shenyangyang4, simmon, Simonas Kazlauskas,
-        Slimane Selyan Amiri, Stefan Agner, Steve Ramage, Susant Sahani, Sven
-        Mueller, Tad Fisher, Takashi Iwai, Thomas Haller, Tom Shield, Topi
-        Miettinen, Torsten Hilbrich, Tyler Hicks, Ulrich Ölmann, Vincent
-        Pelletier, Vinnie Magro, Vito Caputo, Vlad, walbit-de, Weblate, Weblate
-        (bot), Whired Planck, wouter bolsterlee, Xℹ Ruoyao, Yuri Chornoivan, Yu
-        Watanabe, Zach Smith, Zbigniew Jędrzejewski-Szmek, Zmicer Turok, Дамјан
-        Георгиевски
-
-        — Warsaw, 2021-03-11
+        Medri, Daniel Rusek, Dan Sanders, Dan Streetman, Darren Ng, David
+        Edmundson, Deepak Rawat, Devon Pringle, Dmitry Borodaenko, dropsignal,
+        Einsler Lee, Endre Szabo, Evgeny Vereshchagin, Fabian Affolter, Felipe
+        Borges, feliperodriguesfr, Felix Stupp, Florian Hülsmann, Florian
+        Klink, Florian Westphal, Franck Bui, Frantisek Sumsal, Gablegritule,
+        Gaël PORTAY, Gaurav, Giedrius Statkevičius, Greg Depoire--Ferrer,
+        Gustavo Costa, Hans de Goede, Hela Basa, heretoenhance, Iago López
+        Galeiras, igo95862, Ilya Dmitrichenko, Jameer Pathan, Jan Tojnar,
+        Jiehong, Jinyuan Si, Joerg Behrmann, John Slade, Jonathan G. Underwood,
+        Jonathan McDowell, Josh Triplett, Joshua Watt, Julia Cartwright, Julien
+        Humbert, Kairui Song, Karel Zak, Kevin Backhouse, Kevin P. Fleming,
+        Khem Raj, Konomi, krissgjeng, l4gfcm, Lajos Veres, Lennart Poettering,
+        Luca Boccassi, Luca BRUNO, Lucas Werkmeister, Luka Kudra, Luna
+        Jernberg, Marc-André Lureau, Martin Wilck, Matthias Klumpp, Matt
+        Turner, Michael Marley, Michal Fabik, Michał Kopeć, Michal Koutný,
+        Michal Sekletár, Mike Gilbert, milovlad, moson-mo, Nick, nihilix-melix,
+        Oğuz Ersen, Ondrej Mosnacek, pali, Pavel Hrdina, Pavel Sapezhko, Perry
+        Yuan, Peter Hutterer, Pierre Dubouilh, Piotr Drąg, Richard Laager,
+        rnhmjoj, RussianNeuroMancer, Sebastiaan van Stijn, Sergey Bugaev,
+        shenyangyang4, simmon, Simonas Kazlauskas, Slimane Selyan Amiri, Stefan
+        Agner, Steve Ramage, Susant Sahani, Sven Mueller, Tad Fisher, Takashi
+        Iwai, Thomas Haller, Tom Shield, Topi Miettinen, Torsten Hilbrich,
+        Tyler Hicks, ulf-f, Ulrich Ölmann, Vincent Pelletier, Vinnie Magro,
+        Vito Caputo, Vlad, walbit-de, Weblate, Weblate (bot), Whired Planck,
```

Comment:
Does it make sense to list Weblate and Weblate (bot) here?

Reply:
not really i guess. it' generated by a script

Reply:
Better to treat the bots nicely. You never know when a robot uprising might come :-)